elasticjob-api/src/main/java/org/apache/shardingsphere/elasticjob/api/JobConfiguration.java

```diff
@@ -106,6 +111,8 @@ public static Builder newBuilder(final String jobName, final int shardingTotalCo
         private String jobExecutorServiceHandlerType;
         
         private String jobErrorHandlerType;
+    
+        private final Collection<String> jobListenerTypes = new ArrayList<>();
```

Comment:
Collection is unordered. Consider HashSet or other?

Reply:
Collection is better, HashSet is not ordered too.